gatsby/src/pages/index.tsx

```diff
@@ -1,105 +1,223 @@
-				<GridItem cols={12} sm={6} md={4}>
-					<Panel variant="impact-alt">
-						<h2 className="h3">Drugs A&nbsp;to&nbsp;Z</h2>
-						<p>Browse the list of drug monographs, arranged alphabetically.</p>
-						<Button to="/drugs/" variant="cta" elementType={Link}>
-							Browse drugs
-						</Button>
-					</Panel>
-				</GridItem>
-				<GridItem cols={12} sm={6} md={4}>
-					<Panel variant="primary">
-						<h2 className="h3">Interactions</h2>
+			<div className={styles.homeWrapper}>
+				<SEO />
+				<Hero isBNF={isBNF} />
+				<Grid gutter="loose" className={styles.grid}>
+					<GridItem md={6} cols={12} className={styles.drugsColumn}>
+						<h2>Drugs</h2>
 						<p>
-							Browse the list of drug interactions, arranged alphabetically.
+							Drug monographs describe the uses, doses, safety issues, medicinal
+							forms and other considerations involved in the use of a drug.
 						</p>
-						<Button to="/interactions/" variant="inverse" elementType={Link}>
-							Browse interactions
-						</Button>
-					</Panel>
-				</GridItem>
-				<GridItem cols={12} sm={6} md={4}>
-					<Panel variant="impact-alt">
-						<h2 className="h3">Treatment summaries</h2>
-						<p>
-							Browse the list of treatment summaries, arranged alphabetically.
-						</p>
-						<Button
-							to="/treatment-summaries/"
-							variant="secondary"
+						<h3 className="h4">Browse drugs by A-Z</h3>
+						<Alphabet
+							chunky
+							data-tracking="drugs-a-to-z"
 							elementType={Link}
+							className={styles.alphabet}
 						>
-							Browse treatment summaries
-						</Button>
-					</Panel>
-				</GridItem>
-			</Grid>
-
-			<p>
-				<Link to="/about/">About</Link>
-			</p>
-			<p>
-				<Link to="/dental-practitioners-formulary/">
-					Dental Practitioners&apos; Formulary
-				</Link>
-			</p>
-			<p>
-				<Link to="/medical-devices/">Medical devices</Link>
-			</p>
-			<p>
-				<Link to="/wound-management/">
-					Wound management products and elasticated garments
-				</Link>
-			</p>
-			<p>
-				<Link to="/nurse-prescribers-formulary/">
-					Nurse Prescribers&apos; Formulary
-				</Link>
-			</p>
-			<p>
-				<Link to="/medicines-guidance/">Medicines guidance</Link>
-			</p>
+							{alphabet.map((letter) => (
+								<Letter key={`alphabet_${letter}`} to={`/drugs/#${letter}`}>
+									{letter.toUpperCase()}
+								</Letter>
+							))}
+						</Alphabet>
+					</GridItem>
+
+					<GridItem md={6} cols={12} className={styles.featureColumn}>
+						<div>
+							<h2>Treatment summaries</h2>
+							<p>Browse an A-Z list of treatment summaries covering:</p>
+							<ul>
+								<li>drug use related to a particular body system</li>
+								<li>drug management or treatment of common conditions</li>
+								<li>comparisons between groups of drugs.</li>
+							</ul>
+							<Button
+								to="/treatment-summaries/"
+								variant="primary"
+								elementType={Link}
+							>
+								View treatment summaries A-Z
```

Comment:
Can we change this text from `A-Z` to `A to Z`. See this note from https://service-manual.nhs.uk/design-system/patterns/a-to-z-page:

> We write "A to Z" not "A-Z" because this is generally clearer for people who use screen readers. Some screen readers ignore the dash and read "A-Z" as "Ay zed".

And same comment for all other uses of "A-Z"